content/workshops/ansible_tower/_index.adoc

```diff
@@ -0,0 +1,95 @@
+---
+title: Ansible Tower Workshop
+menu:
+  main:
+    parent: workshops
+    pre: fa fa-magic
+---
+
+:domain_name: redhatgov.io
+:workshop_prefix: company-name
+:tower_url: https://{workshop_prefix}.*student_number*.{domain_name}
+:ssh_url: https://{workshop_prefix}.*student_number*.{domain_name}/wetty/ssh/ec2-user
+
+:icons: font
+:iconsdir: http://people.redhat.com/~jduncan/images/icons
+:imagesdir: /workshops/ansible_tower/images
+
+
+= Automation for Everyone
+
+
+
+.Today's Agenda
+
+- Arrivals
+- Introductions / Agenda
+- Presentation
+- Setup
+- Exercise 1.0 - Ad-Hoc Commands
+- Break
+- Exercise 1.1 - Writing Your First Playbook
+- Exercise 1.2 - Running Your First Playbook
+- Exercise 1.3 - Installing Ansible Tower
+- Exercise 1.4 - Configuring Ansible Tower
+- Exercise 1.4 - Creating and Running a Job Template
+- Wrap Up
+
+
+
+.Before We Get Started
+
+[.lead]
+Overview
+
+Today is meant for anyone who has any exposure to Ansible, whether you have used it or not. We are going to start with a short overview (yes, slideware) and then we'll get into the lab as soon as possible. That is where we will spend most of our time.
+
+[.lead]
+Your Responsibilities
+
+* Have a discussion. This will be boring if it's just us up here talking for over 4 hours.
+* Participate. We are going to cut you loose with Ansible here in just a little while. Have questions. Have opinions.
+
+Hopefully you have your laptop with you. If not, please find a shoulder-surfing buddy. See? Not only can we dig into Ansible but you can make a new friend!
+
+[.lead]
+Conventions Used in This Document
```

Comment:
Change this to be a 2nd level heading for same reason as above.

`== Conventions Used in This Document`